src/transformState.js

```diff
@@ -6,6 +6,31 @@
  */
 function transformState(state, actions) {
   // write code here
+  for (const action of actions) {
+    let whatToDo = action.type;
+
+    switch (whatToDo) {
+      case 'addProperties':
+        Object.assign(state, action.extraData);
+        break;
+
+      case 'removeProperties':
+        action.keysToRemove.forEach(function(keyToRemove) {
+          delete state[keyToRemove];
+        });
```

Comment:
its better to use arrow function
```suggestion
        action.keysToRemove.forEach((keyToRemove) => {
          delete state[keyToRemove];
        });
```